pints/_mcmc/__init__.py

```diff
@@ -1102,3 +1102,76 @@ def mcmc_sample(log_pdf, chains, x0, sigma0=None, method=None):
     """
     return MCMCController(    # pragma: no cover
         log_pdf, chains, x0, sigma0, method=method).run()
+
+
+def initialise_finite(log_pdf, chains, random_sampler=None, max_n_tries=50,
+                      parallel=False, n_workers=None):
+    """
+    Draws parameter values from a given sampling distribution until either
+    finite values for each of ``chains`` have been generated or the number of
+    attempts exceeds ``num_tries``.
+
+    If log_pdf is of :class:`LogPosterior`, then the
+    `log_pdf.log_prior().sample` method is used for initialisation, although
+    this is overruled by `random_sampler` if it is supplied.
+
+    Parameters
+    ----------
+    log_pdf : pints.LogPDF
+        A :class:`LogPDF` function that evaluates points in the parameter
+        space.
+    chains : int
+        The number of initial values to generate.
+    random_sampler : stochastic function
```

Comment:
Mention it's optional if the log_pdf is a posterior?

Reply:
Done.